resources/monitoring/charts/alert-rules/templates/kyma-rules.yaml

```diff
@@ -11,16 +11,16 @@ metadata:
     app: kyma.rules
 spec:
   groups:
-  - name: pod-not-running-rule
-    rules:
-    - alert: SystemPodNotRunning
-      expr: sum(kube_pod_container_status_running { namespace=~"kyma-.*|kube-.*|istio-.*|natss", pod!~"(test.*)|((dummy|sample)-.*)|(.*(docs|backup|test)-.*)|((oct-tp-testsuite-all)-.*)|(.*-(tests|dummy))" } == 0 )by (pod,namespace) * on(pod, namespace) (kube_pod_status_phase{phase="Succeeded"} != 1)
-      for: 60s
-      labels:
-        severity: critical
-      annotations:
-        description: "{{`{{ $labels.namespace }}`}}/{{`{{ $labels.pod }}`}} is not running"
-        summary: "{{`{{ $labels.pod }}`}} is not running"
+  #- name: pod-not-running-rule
```

Comment:
Checking https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/alertmanager/templates/alertmanager.config.yaml you can just change the severity label and no notification is happening anymore.

Reply:
With that alert is still active and you can see on the cluster if it is firing or not, but it is not notifying anymore.

Reply:
Thanks for the suggestion, sounds good! I've applied the suggested change.